AppCreationScripts/apps.json

```diff
@@ -36,11 +36,12 @@
         "settingFile": "/JavaScriptSPA/config.js", 
             "replaceTokens": 
               {
-            "clientId": "Enter_the_Application_Id_Here",
-            "authority": "Enter_the_Cloud_Instance_Here/Enter_the_Tenant_Info_Here",
-            "redirectUri": "Enter_the_Redirect_Uri_Here",
-            "graphMeEndpoint": "Enter_the_Graph_Endpoint_Here/v1.0/me",
-            "graphMailEndpoint": "Enter_the_Graph_Endpoint_Here/v1.0/me/messages"
+              "appId": "Enter_the_Application_Id_Here",
+              "tenantId": "Enter_the_Tenant_Info_Here",
+              "authorityEndpointHost": "Enter_the_Cloud_Instance_Id_Here",
+              "redirectUri": "Enter_the_Redirect_Uri_Here",
+              "msgraphEndpointHost": "Enter_the_Graph_Endpoint_Here/v1.0/me",
+              "msgraphEndpointHost:": "Enter_the_Graph_Endpoint_Here/v1.0/me/messages"
```

Comment:
Repetitive. Please remove.

Reply:
same, outdated file. recent pushes fixed this.